discovery_engine_core/ai/ai/src/coi/key_phrase.rs

```diff
@@ -213,7 +213,6 @@ fn unify(
     let mut candidates = candidates
         .iter()
         .filter_map(|candidate| {
-            let candidate = clean_query(candidate);
```

Comment:
shouldn't we make this conditional based on weather the mlt endpoint is used?

Reply:
We have no idea if that is used or not here. There is no plan to revert the use of mlt.
We can always do `clean_query` when generating the filter and that is probably the correct place where to do it.